wdfn-server/waterdata/templates/base.html

```diff
@@ -5,12 +5,8 @@
     <link rel="stylesheet" href="{{ 'main.css' | asset_url }}">
 {% endblock %}
 
-{% block page_script %}
-    <script src="{{ 'scripts/wdfnviz.js' | asset_url }}"></script>
-{% endblock %}
-
 {% block body %}
-
+    <script async src="{{ 'scripts/wdfnviz.js' | asset_url }}"></script>
```

Comment:
Confused as to why does having this outside the script block helps?

Reply:
The USWDS assumes that the body tag exists when it starts to execute it's javascript (in other words it doesn't wait for everything to load).

Reply:
To clarify, I am referring to the html <body> tag

Reply:
Ah, gotcha.

Reply:
Also there is some history here. Originally there was one USWDS script to be loaded but this would cause flashing on navigation and accordions in the header. Now there is an uswds-init that must be loaded before the <body> tag. In this last release, they must have updated the main script javascript to assume the <body> tag is already loaded. This requires that the script not be loaded until then. We can also now load async because the init code is in the separate script.